source/FreeRTOS_IP.c

```diff
@@ -1435,8 +1435,8 @@ static eFrameProcessingResult_t prvAllowIPPacket( const IPPacket_t * const pxIPP
                  * broadcast address. */
                 eReturn = eReleaseBuffer;
             }
-            else if( memcmp( ( void * ) &xBroadcastMACAddress,
-                             ( void * ) &( pxIPPacket->xEthernetHeader.xSourceAddress ),
+            else if( memcmp( ( const void * ) ( xBroadcastMACAddress.ucBytes ),
```

Comment:
Are the "( const void * )" casts still necessary?

Reply:
This is to fix one MISRA complain.  memcmp has the first parameter defined as  const void*